iron-form.html

```diff
@@ -130,6 +173,15 @@
         valid = el.validate() && valid;
       }
       return valid;
+    },
+
+    _useValue: function(el) {
+      // Checkboxes and radio buttons should only use their value if they're checked.
+      if (el.type !== 'checkbox' && el.type !== 'radio') {
```

Comment:
Ahh, understood. I thought that all elements in the form were assumed to implement the appropriate behavior.